daily_snapshot_terraform/modules/daily_snapshot/service/init.sh

```diff
@@ -24,31 +24,12 @@ mount -o defaults,nofail,discard,noatime /dev/disk/by-id/scsi-0DO_Volume_snapsho
 chmod 777 forest_db
 mkdir --parents -- "$BASE_FOLDER/forest_db/filops"
 
-# make the script executable
+# make the scripts executable
 chmod +x ./upload_filops_snapshot.sh
+chmod +x ./newrelic_fail2ban.sh
+
+# run new_relic and fail2ban scripts
+./newrelic_fail2ban.sh
```

Comment:
we don't really need to make the script executable. We could just `bash newrelic_fail2ban.sh`